### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by minac**
I have been reading the docs to setup kubecost in our environment and noticed there were a couple of typos. With the help of Grammarly, this PR aims to eliminate them.
I hope it helps!

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
 # Getting Started
 
-Welcome to Kubecost! This page provides commonly used product configurations and feature overviews to help get you up and running after the Kubecost product has been [installed](http://kubecost.com/install). 
-
-__Configuration__
```
Comment (dwbrown2) [inline getting-started.md]: Amazing PR! Thank you. The only thing that I noticed is that in markdown new lines are dropped when you remove trailing double spaces in lists like this.

File: troubleshoot-install.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
-[No persistent volumes available...](#persistent-volume)    
-[Unable to establish a port-forward connection](#port-forward)  
-[FailedScheduling node-exporter](#node-exporter)   
-[No clusters found](#no-cluster)  
+[No persistent volumes available...](#persistent-volume)
```
Comment (dwbrown2) [inline troubleshoot-install.md]: Here's another case.

File: getting-started.md (hunk 1 above)
Comment (minac) [reply getting-started.md]: Hey @dwbrown2! I don't see it in the markdown app I use (Typora), but I re-added the double spaces that my editor took out. Can you please check if this looks good now?

Comment (dwbrown2) [reply getting-started.md]: Looks great @minac! Thanks you again so much. I'll go ahead and merge.

**Pull request merged**